types/highland/index.d.ts

```diff
@@ -1144,7 +1168,7 @@ declare namespace Highland {
 		 * @param {Function} f - the iterator function
 		 * @api public
 		 */
-		each(f: (x: R) => void): void;
+		each(f: (x: R) => void): Pick<Stream<R>, 'done'>;
```

Comment:
I can't find `Pick<T, U>`. Could you tell me where it is defined?

Reply:
It's a new build-in type introduced in TS 2.1
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-1.html

Reply:
Oh, neat! Thanks.